src/Silk.Core/Commands/General/RemindCommand.cs

```diff
@@ -31,6 +40,71 @@ public async Task Remind(CommandContext ctx, TimeSpan time, [RemainingText] stri
             await ctx.RespondAsync($"Alrighty, I'll remind you in {time.Humanize(2, minUnit: TimeUnit.Second)}: {reminder.Pull(..200)}");
         }
         
-        
+        [Command]
+        [Description("Gives you a list of your reminders")]
+        public async Task List(CommandContext ctx)
+        {
+            var reminders = await _reminders.GetRemindersAsync(ctx.User.Id);
+
+            if(reminders is null)
+            {
+                await ctx.RespondAsync("You have no reminders!");
+            }
+            else
+            {
+                string[] allReminders = reminders
+                .Select(r =>
+                {
+                    var s = $"`{r.Id}` → Expiring {r.Expiration.Humanize()}\n";
+                    if(r.ReplyId is not null)
+                    {
+                        s += $"[Referenced message](https://discord.com/channels/{r.GuildId}/{r.ChannelId}/{r.ReplyId})\n";
+                    }
+                    s += $"`{r.MessageContent}`";
+                    return s;
+                }).ToArray();
+
+                if (string.Join('\n', allReminders).Length <= 2048)
+                {
+                    var builder = new DiscordEmbedBuilder()
+                        .WithColor(DiscordColor.Blurple)
+                        .WithTitle($"Reminders for {ctx.User.Username}:")
+                        .WithFooter($"Silk! | Requested by {ctx.User.Id}");
+                    builder.WithDescription(string.Join('\n', allReminders));
+                    await ctx.RespondAsync(builder);
+                }
+                else
+                {
+                    var interactivity = ctx.Client.GetInteractivity();
+
+                    var pages = new List<Page>();
+                    foreach(var reminder in allReminders)
+                    {
+                        pages.Add(new Page("", new DiscordEmbedBuilder()
+                            .WithColor(DiscordColor.Blurple)
+                            .WithTitle($"Reminders for {ctx.User.Username}:")
+                            .WithDescription(reminder)
+                            .WithFooter($"Silk! | Requested by {ctx.User.Id}")));
+                    }
+                    await interactivity.SendPaginatedMessageAsync(ctx.Channel, ctx.User, pages);
+                }
+            }
+        }
+
+        [Command]
+        [Aliases("cancel")]
+        [Description("Removes one of your reminders based on the id given")]
+        public async Task Remove(CommandContext ctx, int id)
+        {
+            if((await _reminders.GetRemindersAsync(ctx.User.Id))?.Any(r => r.Id == id) ?? false)
+            {
+                await _reminders.RemoveReminderAsync(id);
+                await ctx.RespondAsync("Successfully removed reminder.");
+            }
+            else
+            {
+                await ctx.RespondAsync("You don't have any reminder with this id!");
```

Comment:
This should be a bit more user friendly. Something along the lines of “you don’t have any active reminders!” Or “I couldn’t find that reminder!”